autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -383,7 +379,15 @@ def evaluate(self, y_true, y_pred, silent=False, auxiliary_metrics=False, detail
                 y_true_cleaned = y_true_cleaned.fillna(-1)  # map unknown classes to -1
```

Comment:
BTW should we output warning in this case where there's NA labels?

Reply:
Probably not necessary, if it causes a problem with scoring an exception is raised.